### PR TITLE
Harrison react navigation

### LINKED ISSUE #22: add react native navigation
no specific frames. needed for overall functionality.

### PR DESCRIPTION
**Pull request by hpweisberg**
Added react-navigation. Add sample tailwind css to mainBtn. Create a navigation example to the login page when clicking signup/in

### CODE REVIEW

File: App.tsx
Comment (ekelly55) [inline App.tsx:106]: Are you intending to leave this commented out section in the file? I see that it's new, but also commented out.

File: ios/Podfile.lock
Comment (ekelly55) [inline ios/Podfile.lock:704]: Based on the other lines here, should this be capitalized?

**Review by ekelly55 — APPROVED** [review]
Looks good to me with my limited React Native knowledge.

**Comment by ekelly55** [comment]
I reviewed and approved. Waiting for @hpweisberg to review before it can be merged.

File: App.tsx
Comment (hpweisberg) [reply App.tsx:106]: Left there so others can see. Can be deleted if everyone has seen and understands.

File: ios/Podfile.lock
Comment (hpweisberg) [reply ios/Podfile.lock:704]: It should not be, those are auto imported from yarn.

**Pull request merged**
